terminology/terminology_index.adoc

```diff
@@ -0,0 +1,207 @@
+// vim: set syntax=asciidoc:
+[[Terminology]]
+== Terminology
+:data-uri:
+:homepage https://github.com/projectatomic/container-best-practices:
+
+=== Dictionary
+
+When discussing containerization, it’s important to have a solid grasp on the related vocabulary. One of the challenges people have is that many of the following terms
+are used interchangeably.It can be confusing, especially for newcomers.
+
+The goal of this section is to clarify these terms, so that we can speak the same language.
+
+==== Container Image
+
+Container image is a binary that includes all of the requirements for running a container, as well as metadata describing its needs and capabilities. You can think of it as a packaging technology. Containers only have access to resources defined in the image,
+unless you give the container additional access when creating it.
+
+==== Container
+
+A container is an instance of an image. Containers are an abstraction at the application layer that packages code and dependencies together.
```

Comment:
Would you mind changing this to...
```
A container is composed of two things: a writable filesystem layer on top of a container image, and a tradiional linux process.
```

Reply:
Mine explanation should be more precise, your explanation is more user friendly :D